### ISSUE TITLE
[BUG]: local host hight latency and no boat sprite

### ISSUE THREAD

**Issue by stephane-archer** [TheHunterManX/GBA-PK-multiplayer#77]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the bug

while in a boat the player is shown not in a boat, also there is a large latency between players but we are on the same computer playing using localhost
https://www.youtube.com/live/WdknJRfhsvs?si=EQO33lm_v-Rr_S2R&t=3551

### Steps To Reproduce

just take the boat at the same time as another player on Emerald.

**Comment by TheHunterManX** [comment]
I don't believe it is a latency issue, but rather an animation issue. And yes, I did not account for the boat. I will add this to fixes needed to be done.
